script.js

```diff
@@ -1,5 +1,203 @@
-function runProgram(){
+function displayPresInformation(president){
+    console.log(president);
 
+    let CardContainer = document.querySelector(".bottom-container")
+    let newCard = createCard(president);
+    CardContainer.appendChild(newCard);
+
+}
+
+function displayPresName(president){
+    console.log("function displayPresName running");
+    let NameContainer = document.querySelector(".bottom-container");
+    let NameCard = createNameCard(president);
+    NameContainer.appendChild(NameCard);
+}
+
+function createNameCard(president){
+    let nameCard = document.createElement('p');
+    nameCard.classList.add('.info-container');
+
+    let PresName = document.querySelector('div');
+    PresName.classList.add('card-body');
+
+    let name = document.createElement('p');
+    name.textContent = `Name: ${president.name}`;
+    name.classList('card-title');
+    
+    PresName.appendChild(name);
+    nameCard.appendChild(nameCard);
+    
+    return PresCard;
+}
+
+
+function createCard(president){
+    let PresCard = document.createElement('p');
+    PresCard.classList.add('info-container');
+
+    //create president detail div
+    let PresDetails = document.createElement('div');
+    PresDetails.classList.add('card-body');
+
+    //making the title element and name
+    let PresTitle = document.createElement('p');
+    PresTitle.textContent = `Name: ${president.name}`;
+    PresTitle.classList.add('card-title');
+    PresDetails.appendChild(PresTitle);
```

Comment:
Before adding any 'sub elements' of the card, do a check to see if you need to add it: 
```js
    if(filterValue === 'Name' || filterValue === 'all'){
        PresDetails.appendChild(PresTitle);
    }
    
 ```